saw-core/src/Verifier/SAW/SharedTerm.hs

```diff
@@ -745,14 +747,17 @@ reducePi sc t arg = do
     _ ->
       fail $ unlines ["reducePi: not a Pi term", showTerm t']
 
+-- | Copmute the type of a global variable.
 scTypeOfGlobal :: SharedContext -> Ident -> IO Term
 scTypeOfGlobal sc ident =
   defType <$> scRequireDef sc ident
 
+-- | Compute the type of a datatype.
 scTypeOfDataType :: SharedContext -> Ident -> IO Term
 scTypeOfDataType sc ident =
   dtType <$> scRequireDataType sc ident
 
+-- | Compute the type of an element of the @llvm.global_ctors@ array.
```

Comment:
This operation has nothing to do with LLVM; rather, given the name of a constructor from a saw-core `data` declaration, this will compute its type.

Reply:
Ha, this is what I get for trying to piece together what everything is for by chasing names through files :)